<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QMainWindow" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreditCalc (by Burshlat)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(255, 219, 139);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>135</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>330</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Monthly payment:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePercent">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>135</y>
      <width>125</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>451</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="tableLayout">
     <item>
      <widget class="QScrollArea" name="saTableScroll">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>447</width>
          <height>437</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>0</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMonthResult">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>330</y>
      <width>221</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>380</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Loan overpayment:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Rub.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Rub.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSumResult">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>430</y>
      <width>221</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Rub.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>0</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Repayment amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>35</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Loan amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>135</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Interest rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTerm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>85</y>
      <width>125</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>430</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Total payout:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>35</y>
      <width>125</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>0</y>
      <width>131</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Repayment dates</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>202</y>
      <width>251</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Type of monthly payments:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>190</y>
      <width>221</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbAnnu">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Annuities</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDiffer">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Differentiated</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>85</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbMonth">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Months</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbYear">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Years</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Loan term:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResult">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>265</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>35</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black; 
}</string>
    </property>
    <property name="text">
     <string>Rub.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOverpay">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>380</y>
      <width>221</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
